Release v0.11.0 (#90)
chrovis/varity: CHANGELOG.md

# Changelog

## [0.11.0] - date TBD

### Fixed

> - Return protein-hgvs nil when variants overlapping exon/intron boundaries. [#89](https://github.com/chrovis/varity/pull/89)

Reviewer: ```suggestion
- Return protein HGVS as `nil` if a variant overlaps exon-intron boundaries. [#89](https://github.com/chrovis/varity/pull/89)
```
